packages/ui/src/machines/authenticator/actors/signUp.ts

```diff
@@ -164,28 +170,20 @@ export function signUpActor({ services }: SignUpMachineOptions) {
                   entry: ['sendUpdate'],
                   on: {
                     SUBMIT: { actions: 'handleSubmit', target: 'validate' },
-                    FEDERATED_SIGN_IN: 'federatedSignIn',
-                  },
-                },
-                federatedSignIn: {
-                  entry: ['sendUpdate', 'clearError'],
-                  invoke: {
-                    src: 'federatedSignIn',
-                    onError: { actions: 'setRemoteError' },
                   },
                 },
                 validate: {
                   entry: 'sendUpdate',
                   invoke: {
                     src: 'validateSignUp',
                     onDone: {
-                      target: 'signUp',
+                      target: 'handleSignUp',
```

Comment:
Renamed to `handleSignUp` to help differentiate between the parent `signUp` state